### PR TITLE
Fix promise check

### PR DESCRIPTION
**Pull request by maltsev**
Related PR: #112

### CODE REVIEW

**Comment by maltsev** [comment]
I'll appreciate if you make a release with this bugfix soon. One of my plugins is broken because of this issue.

**Comment by zxqfox** [comment]
:+1:

File: lib/posthtml.js [hunk 1]
```diff
@@ -136,8 +136,10 @@ module.exports = function(plugins) {
  * @returns {Boolean}
  */
 function isPromise(p) {
-    return typeof Promise === 'function' && p instanceof Promise ||
-           typeof p === 'function' && p.then && p.catch;
+    // Source: https://github.com/then/is-promise/blob/master/index.js
```
Comment (voischev) [inline lib/posthtml.js]: remove this line :)

**Comment by voischev** [comment]
@maltsev :+1: Thx!

**Pull request merged**